atlasdb-dropwizard-bundle/versions.lock

```diff
@@ -763,35 +763,36 @@
             ]
         },
         "org.eclipse.jetty:jetty-continuation": {
-            "locked": "9.2.17.v20160517",
+            "locked": "9.4.0.v20161208",
```

Comment:
I think we probably want to pin the Jetty version bump to the `timelock-server` module similar to DW version, as bumping these will likely cause downstream version dependency pains.